src/components/settings/NumberSetting.tsx

```diff
@@ -39,12 +39,15 @@ type BaseProps = {
     showSlider?: never;
     disabled?: boolean;
     listItemTextSx?: SxProps<Theme>;
+    handleLiveUpdate?: (value: number) => void;
```

Comment:
looking at it again, do we even need this new callback, couldn't it just be an additional parameter (e.g. `persist`) of `handleUpdate` that is only true if the ok button got clicked?

Reply:
That feels like a weird thing to add to the generic `NumberSetting` component. Persistence isn't really something this component should need to worry about, no?

Reply:
maybe `persist` was a bad name, more like `okClicked`, but nevermind me, I think having 2 callbacks is easier to use